index.js

```diff
@@ -13,65 +14,85 @@ if (require.main === module) {
 }
 
 function frontendDependencies(workDir) {
+
+    // prepare everything
     if (!workDir) {
         workDir = process.cwd();
     }
-
     var packageJson = require(path.join(workDir, "package.json"));
-
     if (!packageJson.frontendDependencies) {
         fail("No 'frontendDependencies' key in package.json");
     }
-
     if (!packageJson.frontendDependencies.packages) {
-        fail("No 'frontendDependencies.packages' key in package.json");
+        fail("No 'frontendDependencies.packages' in package.json");
+    }
+
+
+    // maybe remove this code in later versions
+    if (packageJson.frontendDependencies.packages.constructor === Array) {
+        fail("You have an old version of this package: Update to version 0.5.0 and change the syntax in your package.json");
```

Comment:
If the user is running this code, that means it is already updated, so printing a message like "Please, update your package.json frontendDependencies format to  > 1.0.0 syntax as explained at https://github..." would be enough?

Reply:
yes, my mistake